Gemfile

```diff
@@ -28,6 +28,9 @@ group :doc do
   gem 'sdoc', require: false
 end
 
+# Debugger
+gem 'byebug'
```

Comment:
I don't have a strong opinion pro or con byebug, but I think we should only add this to test and development environment